model/src/test/test_datasets.py

```diff
@@ -13,7 +15,7 @@ def test_loading_wastewater():
     assert df.shape == (635, 14)
 
     testing.assert_approx_equal(df["daily_hosp_admits"].mean(), 12.8888, 3)
-    testing.assert_approx_equal(df["lod_sewage"].mean(), 3.841025, 3)
+    testing.assert_approx_equal(df["load_sewage"].mean(), 3.841025, 3)
```

Comment:
LOD is OK: https://github.com/CDCgov/wastewater-informed-covid-forecasting/blob/61af2bbbd0cc179460b3b06c4fa209b3a50f6101/cfaforecastrenewalww/R/data.R#L22. Need to update the other two references.

Reply:
Addressed in f71bcda